test/test_torch.py

```diff
@@ -4170,7 +4170,7 @@ def test_randint_inference(self):
         for args in [(3,), (1, 3)]:  # (low,) and (low, high)
             self.assertIs(torch.int64, torch.randint(*args, size=size).dtype)
             self.assertIs(torch.int64, torch.randint(*args, size=size, layout=torch.strided).dtype)
-            self.assertIs(torch.int64, torch.randint(*args, size=size, generator=torch.default_generator).dtype)
+            self.assertIs(torch.int64, torch.randint(*args, size=size, generator=torch.default_generator[0]).dtype)
```

Comment:
Hmmmmm... did you mean to also turn CPU default generator into a tuple? That looks extremely suspect to me...

Reply:
Yeah, I don't think this should be a tuple.